Branch.podspec

```diff
@@ -23,5 +23,5 @@ Use the Branch SDK (branch.io) to create and power the links that point back to
   s.requires_arc = true
 
   s.source_files = "Branch-SDK/Branch-SDK/*.{h,m}"
-  s.frameworks = 'AdSupport', 'CoreTelephony'
+  s.frameworks = 'AdSupport', 'CoreTelephony', 'CoreSpotlight', 'MobileCoreServices'
```

Comment:
What happens if someone drops the framework in and leaves off these frameworks - will there be build errors?